portal-ui/src/screens/Console/Logs/ErrorLogs/ErrorLogs.tsx

```diff
@@ -170,38 +231,96 @@ const ErrorLogs = ({
+                  disabled={loadingNodes}
+                  input={<SelectStyled />}
                 >
-                  <Table aria-label="collapsible table">
-                    <TableBody>
-                      {filteredMessages.map((m) => {
-                        return <LogLine key={m.key} log={m} />;
-                      })}
-                    </TableBody>
-                  </Table>
-                </TableContainer>
-              </div>
+                  <MenuItem
+                    value={selectedNode}
+                    key={`select-node-default`}
+                    disabled={true}
+                  >
+                    Select Node
+                  </MenuItem>
+                  {nodes.map((aNode) => (
+                    <MenuItem value={aNode} key={`select-node-name-${aNode}`}>
+                      {aNode}
+                    </MenuItem>
+                  ))}
+                </Select>
+              </FormControl>
             ) : (
-              <Box
-                sx={{
-                  padding: "15px",
-                  display: "flex",
-                  alignItems: "center",
-                  justifyContent: "center",
-                }}
-              >
-                {filter.trim().length
-                  ? `No matching logs for "${filter.trim()}".`
-                  : "No logs to display."}
-              </Box>
+              <h3> Loading nodes</h3>
             )}
+
+            <div className={classes.checkBoxLabel}>Log type to display:</div>
+            <div className={classes.midColumnCheckboxes}>
+              <CheckboxWrapper
+                checked={allTypes}
+                id={"all_calls"}
+                name={"all_calls"}
+                label={"All"}
+                onChange={(item) => {
+                  setAllTypes(!allTypes);
+                  item.target.checked ? setLogType("all") : setLogType("");
+                }}
+                value={"all"}
+                disabled={false}
+              />
+              <CheckboxWrapper
+                checked={logType === "application" || allTypes}
+                id={"application_type"}
+                name={"application_type"}
+                label={"Application"}
+                onChange={(item) => {
+                  setAllTypes(false);
+                  item.target.checked
+                    ? setLogType("application")
+                    : setLogType("");
+                }}
+                value={"s3"}
+                disabled={false}
+              />
+              <CheckboxWrapper
+                checked={logType === "minio" || allTypes}
+                id={"minio_type"}
+                name={"minio_type"}
+                label={"Minio"}
+                onChange={(item) => {
+                  setAllTypes(false);
+                  item.target.checked ? setLogType("minio") : setLogType("");
+                }}
+                value={"Minio"}
```

Comment:
Another minor issue, should we change the `label` and `value` here from `Minio` to `MinIO`?

```suggestion
                label={"MinIO"}
                onChange={(item) => {
                  setAllTypes(false);
                  item.target.checked ? setLogType("minio") : setLogType("");
                }}
                value={"MinIO"}
```